src/main/resources/static/css/styles.css

```diff
@@ -1,13 +1,45 @@
 #forecast-screen {
-  margin-top: 10px;
-  margin-left: 10px;
-  margin-right: 10px;
-  margin-bottom: 10px;
+	margin: 10px;
 }
 
 #title {
-  margin-top: 15px;
-  margin-left: 5px;
-  margin-right: 5px;
-  margin-bottom: 15px;
+	margin: 15px 5px;
 }
+
+.pagination_section {
```

Comment:
The list for when you display the hourly temperatures can get a little long. Perhaps consider implementing pagination for it. Here are some styles to get you started.